wall_e/extensions/reaction_role.py

```diff
@@ -533,6 +556,41 @@ async def edit(self, ctx: commands.Context, message_id):
         for emoji in add_emojis:
             await message.add_reaction(emoji)
 
+    async def delete(self, ctx: commands.Context, message_id):
+        """Deletes a reaction role"""
+        self.logger.info('[ReactionRole delete()] Deleting reaction role')
+
+        if int(message_id) not in ReactionRole.l_reaction_roles.keys():
+            self.logger.info('[ReactionRole delete()] reaction role not found')
+            await ctx.send('Can\'t find that reaction role. Make sure you have the correct message id')
+            return
+
+        channel_id = await ReactRole.get_channel_id_by_message_id(message_id)
+        channel = self.guild.get_channel(channel_id)
+        if channel is None:
+            await ctx.send(f'Channel with id {channel_id} not found. Channel might have been deleted.')
+            self.logger.info('[ReactionRole delete()] channel not found')
+            await ctx.message.add_reaction('\N{CROSS MARK}')
+            return
+        try:
+            message = await channel.fetch_message(message_id)
+        except discord.errors.NotFound:
+            await ctx.send(f'Message with id {message_id} not found. The message might have been deleted.')
+            self.logger.info('[ReactionRole delete()] message not found')
+            await ctx.message.add_reaction('\N{CROSS MARK}')
+            return
+
+        # update local
+        del ReactionRole.l_reaction_roles[int(message_id)]
+
+        # update db
+        await ReactRole.delete_react_role_by_message_id(message_id)
+
+        # delete message
+        await message.delete()
+        self.logger.info('[ReactionRole delete()] message deleted')
+        await ctx.message.add_reaction('\N{WHITE HEAVY CHECK MARK}')
+
     @commands.command(aliases=['rr'])
     @commands.has_any_role("Minions", "Moderator")
```

Comment:
please convert this to a slash command to keep in line with the initiative started [here](https://github.com/CSSS/wall_e/issues/417)

Reply:
[its a really simple change](https://github.com/CSSS/wall_e/wiki/wall_e-Discord.py-Coding-Tips#21-slash-command-help-message-configurations), you can just do it in this one.